argnorm/drug_categorization.py

```diff
@@ -50,11 +81,21 @@ def drugs_to_drug_classes(drugs_list: List[str]) -> List[str]:
     for drug in drugs_list:
         drug_instance = ARO[drug]
         drug_instance_superclasses = list(drug_instance.superclasses())
-        superclasses_len = len(drug_instance_superclasses)
+        drug_classes += navigate_superclasses(drug_instance_superclasses)
+
+        has_part_nodes = drug_instance.relationships.get(has_part_rel, [])
+        if has_part_nodes:
+            for hast_part_node in has_part_nodes:
+                hast_part_node_superclasses = list(hast_part_node.superclasses())[1:]
+
+                for super_class in hast_part_node_superclasses:
+                    super_class_categories = list(super_class.superclasses())
+                    drug_classes += navigate_superclasses(super_class_categories)
+
+                if hast_part_node.relationships.get(is_small_inhibitor_molecule_rel, []):
```

Comment:
I need to go back to my notes, but I thought we had decided **against** this!

Reply:
Oh, did we not agree that we would ignore antibiotic mixture in all cases except for when we have antibiotic + adjuvant (e.g. https://card.mcmaster.ca/ontology/40955)? This was to make it a bit clearer that it wasn't a mixture of 2 antibiotics such as gentamicin (https://card.mcmaster.ca/ontology/46133), but rather and a beta-lactamase inhibitor was also present.

Reply:
I thought that the decision was that cases like beta-lactam+beta-lactamase inhibitor should be treated as _beta-lactam_.

Reply:
Ah. I think this approach would be better considering we don't report 'antibiotic mixture' for the other mixtures. I'll remove the is_small_inhibitor_molecule_rel then.